rfcs/0006-or-condition.md

```diff
@@ -10,10 +10,18 @@
 This proposal will suggest adding an OR clause to queries that gather the contract to patient mapping in order
 to improve efficiency on both the client and server. 
 
+## Status
+[Status]: #status
+
+* Implementation Status: Done
```

Comment:
This is the only one that I am unsure about. I looked in the code to try to find evidence that this is possible now but I cannot find it (doesn't mean that it is not there though). Also I cannot get to the jira ticket below. Looping in @karlmdavis in case he knows for sure one way or the other.

Reply:
The Jira ticket just covers the RFC itself, and doesn't link to anything else. I'd ask our AB2D friends if they happen to know. I didn't see any PRs for it, myself, but that's not really useful info.